cpp/src/parquet/page_index.h

```diff
@@ -27,6 +27,7 @@
 namespace parquet {
 
 class EncodedStatistics;
+class SizeStatistics;
 struct PageIndexLocation;
```

Comment:
Can we instead consolidate all these forward declarations in `parquet/type_fwd.h`?